test/specs/collections/Form/Form-test.js

```diff
@@ -44,21 +44,42 @@ describe('Form', () => {
 
   describe('onSubmit', () => {
     it('prevents default on the event when there is no action', () => {
-      const event = { preventDefault: sandbox.spy() }
+      const preventDefault = sandbox.spy()
 
-      shallow(<Form onSubmit={sandbox.spy()} />)
-        .simulate('submit', event)
-
-      event.preventDefault.should.have.been.calledOnce()
+      shallow(<Form onSubmit={_.noop} />)
+        .simulate('submit', { preventDefault })
+      preventDefault.should.have.been.calledOnce()
     })
 
     it('does not prevent default on the event when there is an action', () => {
-      const event = { preventDefault: sandbox.spy() }
+      const preventDefault = sandbox.spy()
 
-      shallow(<Form action='do not prevent default!' onSubmit={sandbox.spy()} />)
+      shallow(<Form action='do not prevent default!' onSubmit={_.noop} />)
         .simulate('submit', event)
+      preventDefault.should.not.have.been.called()
```

Comment:
I don't understand how this is working, `event` is now undefined.

```js
.simulate('submit', event)
```

Shouldn't this be throwing?